modules/data.remote/R/NASA_DAAC_download.R

```diff
@@ -94,37 +94,31 @@ NASA_DAAC_download <- function(ul_lat,
       if (length(granules) == 0) 
         break
       # if it's GLANCE product.
-      # GLANCE product has special data archive.
-      if (doi == "10.5067/MEaSUREs/GLanCE/GLanCE30.001") {
-        granules_href <- c(granules_href, sapply(granules, function(x) {
-          links <- c()
-          for (j in seq_along(x$links)) {
-            links <- c(links, x$links[[j]]$href)
-          }
-          return(links)
-        }))
+      # GLANCE and HLS products have special data archive.
+      if (doi %in% c("10.5067/MEaSUREs/GLanCE/GLanCE30.001", "10.5067/HLS/HLSS30.002")) {
```

Comment:
This approach of manually listing exempted DOIs is not sustainable as the number of exempted DOIs will just keep going up (e.g. you're already missing the landsat half of the HLS band data) and diagnosing which case you are in isn't obvious to users. Is there any reason to not make this version the default? Are there cases where grabbing links beyond links[[1]] will lead to errors or the selection of unwanted files that aren't being filtered out by the later checks (file type, access type, etc.